### ISSUE TITLE
Label in fr:number/fr:currency doesn't point to the input

### ISSUE THREAD

**Issue by avernet** [orbeon/orbeon-forms#2216]
![2015-05-07_1645](https://cloud.githubusercontent.com/assets/106961/7528129/9aebce10-f4d8-11e4-98fb-1d568ba1f24a.png)

This is in violation of WCAG2 1.1.1/1.3.1/3.3.2 (A), and reported as E866 by TotalValidator.

Possible implementations:

- The XBL engine does this for now, which it is not able to do now as the `input` isn't known to the XForms engine.
- We use a `span` and a `aria-labeledby` instead of a `label`; check that TotalValidator is happy with that solution.
